src/AzureExtension/DataManager/AzureDataManager.cs

```diff
@@ -371,6 +371,11 @@ private async Task UpdateDataForQueriesAsync(DataStoreOperationParameters parame
 
                     break;
 
+                case TFModels.QueryType.OneHop:

```

Comment:
I think it should be ok to merge this with `TFModels.QueryType.Tree` case.

```cs
case TFModels.QueryType.OneHop:
case TFModels.QueryType.Tree:
    // Code goes here
    break;
```

Reply:
That was my first solution, but that is [illegal in c#](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0163) and the correct way to do it is explicitly with a goto statement as I have here.

Reply:
Nit: Would be more readable if this case is right above the tree case